docs/book/src/clusterctl/commands/get-kubeconfig.md

```diff
@@ -0,0 +1,18 @@
+# clusterctl get kubeconfig
+
+The `clusterctl get` command prints the kubeconfig of an existing workload cluster into stdout.
+This funcitonality is available in clusterctl version 0.3.9 or newer.
+
+## Usage:
```

Comment:
```suggestion
```
For sake of consistency with others command's doc, we should drop Usage.
Also, I suggest to drop the list of parameters; instead, you can provide an example of calling the command with the `--namespace` flag

Reply:
ack